imio/dms/mail/browser/iconified_category.py

```
@@ -32,6 +33,61 @@
+    """Return ``(css_class, title)`` for the *approved* icon shown, non-clickable,
+       on an outgoing-mail file's view page (viewlet)."""
+    approval = IOMApproval(mail)
+    state = api.content.get_state(mail)
+    uid = fileobj.UID()
+    userid = api.user.get_current().getId()
+    classes = ["iconified-approved"]
+    to_approve = getattr(fileobj, "to_approve", True)
+
+    if approval.is_state_before_approve(state=state):
+        if not to_approve:
+            return " ".join(classes + ["to-approve"]), u"Deactivated for approval"
```

Ruff (0.15.20)

[warning] Consider `[*classes, "to-approve"]` instead of concatenation

Replace with `[*classes, "to-approve"]`

(RUF005)